CHANGELOG.md

```diff
@@ -144,6 +146,32 @@ best long-term interest of our customers.
 as `final`. After the changes in [v1.36.0](#v1360---2022-02), there is no need
 or reason to be extending this class.
 
+**OTHER CHANGES**:
+
+* feat(bigtable): better support for PSC and VPC-SC ([#8458](https://github.com/googleapis/google-cloud-cpp/pull/8458))
+* refactor(bigtable)!: mark InstanceAdminClient as final ([#8449](https://github.com/googleapis/google-cloud-cpp/pull/8449))
+* fix(bigtable): use the appropriate type alias for AsyncRowReader::last_read_row_key_ ([#8437](https://github.com/googleapis/google-cloud-cpp/pull/8437))
+* docs(bigtable): cleanup admin documentation ([#8436](https://github.com/googleapis/google-cloud-cpp/pull/8436))
+* fix(bigtable): ReadRows retries from the `last_scanned_row_key` ([#8423](https://github.com/googleapis/google-cloud-cpp/pull/8423))
+* refactor(bigtable)!: modernize TableAdmin ([#8383](https://github.com/googleapis/google-cloud-cpp/pull/8383))
+* feat(bigtable): support x-goog-user-project ([#8324](https://github.com/googleapis/google-cloud-cpp/pull/8324))
+
+### [IAM](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/iam/README.md)
+
+* docs(iam): added comments on private_key_data ([#8204](https://github.com/googleapis/google-cloud-cpp/pull/8204))
+
+### [Pub/Sub](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/pubsub/README.md)
+
+* feat(pubsub): support `AuthorityOption` ([#8460](https://github.com/googleapis/google-cloud-cpp/pull/8460))
+* feat(pubsub): support x-goog-user-project ([#8456](https://github.com/googleapis/google-cloud-cpp/pull/8456))
+* feat(pubsub): per-call Options for SubscriptionAdminClient ([#8414](https://github.com/googleapis/google-cloud-cpp/pull/8414))
+* feat(pubsub): per-call options in TopicAdminClient ([#8411](https://github.com/googleapis/google-cloud-cpp/pull/8411))
+* fix(pubsub): restore MakeSchemaAdminConnection overload ([#8412](https://github.com/googleapis/google-cloud-cpp/pull/8412))
```

Comment:
I think we can delete this too. I think there was no change from release to release.

Reply:
Done.